R/zzz.R

```diff
@@ -1,12 +1,10 @@
 .onAttach <- function(libname, pkgname) {
-  startup_message()
+  packageStartupMessage(startup_message())
```

Comment:
In that R CMD Check is now not showing a message on that? It's because packageStartupMessage() provides a message but can be suppressed, which is what Check wants.